src/calculateRentalCost.js

```diff
@@ -4,7 +4,23 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const SHORT_TERM = 3;
+  const SHORT_TERM_DISCOUNT = 20;
+  const LONG_TERM = 7;
+  const LONG_TERM_DISCOUNT = 50;
```

Comment:
The names of the constants are descriptive, which is good. However, the daily cost of renting the car is hardcoded in the expression on line 21. This should be extracted into a constant as well to comply with the checklist item 2.